.github/workflows/scenario-test.yml

```diff
@@ -1,6 +1,8 @@
 name: Scenario Tests - DC missions
 
 on:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
As this is a cron job, I would highly recommend to add an automated issue in case that the run fails at some point. You can achieve this via a scripted task:

```yaml
 - name: Create issue
      uses: actions/github-script@v7
      with:
        github-token: ${{ secrets.GITHUB_TOKEN }}
        script: |
          github.rest.issues.create({
            owner: context.repo.owner,
            repo: context.repo.repo,
            title: 'Some Title',
            body: "Some text"
          })
```